frontend/lib/norent/letter-builder/national-metadata.tsx

```diff
@@ -10,17 +10,20 @@ import {
 import { LosAngelesZipCodes } from "../data/la-zipcodes";
 
 type StateLawForBuilderEntry = {
-  linkToLegislation: string;
+  linkToLegislation?: string;
```

Comment:
@sraby this is weird, it seems like `linkToLegislation` is no longer always present, so I made it optional...

Reply:
oh huh, yeah I mean that makes sense since in the `State legislation for letter itself` table that field is not exposed and we don't actually use the link to the legislation in our little KYR page in the letter builder flow